src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -103,13 +122,16 @@ def __call__(
         latents: Optional[torch.FloatTensor] = None,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
+        output_latents: bool = False,
+        run_safety_checker: bool = True,
```

Comment:
FYI, the team has previously shut down PRs that define an API to bypass the safety filter, e.g. #254.

I recommend leaving that out of this PR just so that doesn't get mixed up with the other features you propose here.

Reply:
Sure, I can remove that part if needed 👍

Reply:
removed in https://github.com/huggingface/diffusers/pull/506/commits/12ca96990b0e6d7ef34a0c4bbcd9819ffe2752bd